src/components/crossDevice/ClientSuccess/index.js

```diff
@@ -18,12 +18,12 @@ class ClientSuccess extends Component {
           title={translate('cross_device_return.title')}
           subTitle={translate('cross_device_return.subtitle')}
         />
-        <div>
-          <span className={`${theme.icon}  ${style.icon}`} />
+        <Fragment>
```

Comment:
It seems that you don't need `Fragment` at all because the children are explicitly declared (not within an array map)

Reply:
Ah, I see. So it should only really be used if it is an array map of child elements/components?

Reply:
Actually `<Fragment />` is for situations that you don't know how many nodes it might render, possibly more than one. Eventually `<Fragment />` will be omitted, all inner nodes will be rendered at the same level with the `<Fragment />`.